src/Common/src/CoreLib/System/Decimal.DecCalc.cs

```diff
@@ -690,7 +690,7 @@ private static unsafe int ScaleResult(Buf24* bufRes, uint hiRes, int scale)
                         }
 
                         break;
-                    } // for(;;)
+                    } // for (;;)
```

Comment:
I'm surprised that commented out instructions are also captured.

Reply:
> I'm surprised that commented out instructions are also captured.

They're not.  I also did a search/replace after the fact to catch stragglers like this.